lib/hammer_cli_katello/module_stream.rb

```diff
@@ -0,0 +1,61 @@
+module HammerCLIKatello
+  class ModuleStreamCommand < HammerCLIKatello::Command
+    resource :module_streams
+
+    class ListCommand < HammerCLIKatello::ListCommand
+      extend RepositoryScopedToProduct
+
+      validate_repo_name_requires_product_options(:option_repository_name)
+
+      output do
+        field :id, _("ID")
+        field :name, _("Module Stream Name")
+        field :stream, _("Stream")
+        field :uuid, _("UUID")
+        field :version, _("Version")
+        field :arch, _("Architecture")
+        field :context, _("Context")
+      end
+
+      build_options do |o|
+        o.expand.including(:products)
+      end
+    end
+
+    class InfoCommand < HammerCLIKatello::InfoCommand
```

Comment:
Missing product options for resolving repo ID from repo name.

```sh
$ hammer module-stream info -h
Usage:
    hammer module-stream info [OPTIONS]

Options:
 --id ID                       A module stream identifier
 --name NAME                   Name to search by
 --repository REPOSITORY_NAME  Repository name to search by
 --repository-id REPOSITORY_ID Repository ID
 -h, --help                    Print help

$ hammer module-stream info --name django --product custom --repository modular --organization-id 1
Error: Unrecognised option '--product'.

See: 'hammer module-stream info --help'.
```

Needs similar validation to list.

Reply:
added product options for info and a test as well